src/api/libcellml/generator.h

```diff
@@ -67,6 +74,11 @@ class LIBCELLML_EXPORT GeneratorVariable
     GeneratorVariableImpl *mPimpl;
 };
 
+/**
+ * @brief The Generator class.
+ *
+ * The Generator class is for representing a CellML Generator.
```

Comment:
I don't really think this documentation helps anyone at all.

Reply:
Agreed, I was merely mimicking what is already in the codebase...